src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$column-width: 100px;
+$gap: 1px;
+
+.calendar {
+  display: flex;
+  width: calc((7 * $column-width) + (6 * $gap));
+  flex-wrap: wrap;
+  gap: $gap;
+  box-sizing: border-box;
```

Comment:
Consider using `margin: 0 auto;` on `.calendar` to center it horizontally within its container, instead of using `display: flex;` on the `body`. This would make the layout more flexible and maintainable.